packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,230 @@
+import * as Sentry from '@sentry/browser';
+import { EventProcessor, Hub, Integration, Scope } from '@sentry/types';
+
+/**
+ * NodeJS integration
+ *
+ * Provides a mechanism for NodeJS
+ * that raises an exception for handled, unhandled, timeout and similar times of error
+ * and captures the same in Sentry Dashboard
+ */
+export class AWSLambda implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'AWSLambda';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = AWSLambda.id;
+
+  /**
+   * context.
+   */
+  private _awsContext: {
```

Comment:
Extract this to a separate interface

Reply:
made this change in the new pull request.